images/airflow/2.9.2/python/mwaa/celery/task_monitor.py

```diff
@@ -46,7 +46,7 @@
 # blocks without running out of space.
 BUFFER_SIZE_PER_TASK = 2500
 CELERY_WORKER_TASK_LIMIT = int(
-    os.environ.get("AIRFLOW__CELERY__WORKER_AUTOSCALE", "20,20").split(",")[0]
+    os.environ.get("AIRFLOW__CELERY__WORKER_AUTOSCALE", "80,80").split(",")[0]
```

Comment:
How is this related to the change?

Reply:
It's actually not. These are small improvements that I found while working on the porting code for older and newer versions. Original plan was to add these as a separate PR but this PR was sitting in stale state for quite some time and I merged those changes to this PR to save time. I can remove them if needed.